pkg/examples/extendworkload/new-crd.yaml

```diff
@@ -0,0 +1,213 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.2.5
+  creationTimestamp: null
+  name: applicationconfigurations.ros.alibaba.com
+spec:
+  group: ros.alibaba.com
+  names:
+    kind: ApplicationConfiguration
+    listKind: ApplicationConfigurationList
```

Comment:
I assume that this is the same as OAM appConfig (except the group name), right?

Reply:
The schematic of this thing is same with AppConfig, but I'd suggest define a separate CRD for it (even with similar fields of AppConfig) instead of "reuse".

Reply:
It's just a demo, so I use totally same OAM AppConfig((except the group name. yes)).